styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+}
+
+main {
+  background: #f7f7f9;
+  padding-top: 75px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
```

Comment:
- [ ] Kindly remove the padding left and right so that your design fill the whole page 🙏 

![image](https://user-images.githubusercontent.com/74814780/185521263-079311ba-a764-46dc-be8d-a18f0ed12d0a.png)